tasks/main.yml

```diff
@@ -17,6 +17,23 @@
     group: 'nginx'
     mode: 0775
 
+- name: Rotation for project nginx log file
+  logrotate:
+    name: '{{ application_name }}'
+    path: '/var/log/nginx/{{ application_name }}/*.log'
```

Comment:
Not sure if it's a good idea a wildcard on the line. Though, we wanted to apply log rotate to any log file.
Would it be cool to have it on a `list` (ie [ access.log, error.log ])?

More readable and for maintainability.